vertexai/snippets/text_input.go

```diff
@@ -0,0 +1,61 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package snippets
+
+// [START generativeaionvertexai_gemini_generate_from_text_input]
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/vertexai/genai"
+)
+
+func textInput(w io.Writer, projectID string, location string, modelName string) error {
+	// location := "us-central1"
```

Comment:
These can be local variables rather than args probably, given that they are static in the test.

Reply:
Done